pymysql/converters.py

```diff
@@ -309,13 +309,8 @@ def through(x):
     return x
 
 
-#def convert_bit(b):
-#    b = "\x00" * (8 - len(b)) + b # pad w/ zeroes
-#    return struct.unpack(">Q", b)[0]
-#    
-#     the snippet above is right, but MySQLdb doesn't process bits,
-#     so we shouldn't either
```

Comment:
So is oursql wrong? Am I depending on bad behavior?
On Apr 4, 2016 8:14 PM, "INADA Naoki" <notifications@github.com> wrote:

> In pymysql/converters.py
> <https://github.com/PyMySQL/PyMySQL/pull/441#discussion_r58470079>:
>
> > @@ -309,13 +309,8 @@ def through(x):
> >      return x
> >
> >
> > -#def convert_bit(b):
> > -#    b = "\x00" * (8 - len(b)) + b # pad w/ zeroes
> > -#    return struct.unpack(">Q", b)[0]
> > -#
> > -#     the snippet above is right, but MySQLdb doesn't process bits,
> > -#     so we shouldn't either
>
> Have you read this comment?
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly or view it on GitHub
> <https://github.com/PyMySQL/PyMySQL/pull/441/files/3a27bc21ebc734cac4b5a1574d36e78ff75f7ce4#r58470079>
>